traffic_stats/influxdb_tools/sync/sync_ts_databases.go

```diff
@@ -298,8 +307,9 @@ func syncDailyStat(sourceClient influx.Client, targetClient influx.Client, statN
 
 	db := daily
 	bps, _ := influx.NewBatchPoints(influx.BatchPointsConfig{
-		Database:  db,
-		Precision: "s",
+		Database:        db,
+		Precision:       "s",
+		RetentionPolicy: "indefinite",
```

Comment:
fair enough,  It wasn't included at all before which means it used the default, which is indefinite, but since it is now there I figured it was best to use the variable.  I can live with this for now until we make the whole thing more dynamic.